client/src/Main.elm

```diff
@@ -1517,6 +1622,11 @@ appendInsertToLevel1 l0List newL0 =
                 l0List
 
 
+saveGenbank : String -> String -> Cmd Msg
+saveGenbank vectorName content =
+    File.Download.string (String.append vectorName ".gbk") "text/genbank" content
```

Comment:
MIME type for Genbank seems to be:
* `chemical/x-genbank`
  * https://www.ch.ic.ac.uk/chemime/
  * https://bugs.freedesktop.org/show_bug.cgi?id=335
  * https://mimeapplication.net/chemical-x-genbank
  * https://en.wikipedia.org/wiki/Chemical_file_format#The_Chemical_MIME_Project
  * https://github.com/dleidert/chemical-mime/blob/master/src/chemical-mime-database.xml.in#L736=

Reply:
Also note the extension used, `.gb` or `.gen`